batch_convert.qml

```diff
@@ -269,14 +275,16 @@ MuseScore {
             } // Column
         } // inFormats
         ColumnLayout {
-            Layout.alignment: Qt.AlignTop | Qt.AlignRight
+            Layout.alignment: Qt.AlignTop | Qt.AlignLeft
```

Comment:
Not sure whether this makes any difference, at least I don't see any

Reply:
Indeed. It had an effect earlier in my GUI tweaking  process. But now, its effect is no longer visible. I'll revert it.